try{
let x=undefined;
console.log(x[0]);

}
catch(e){
    console.log("Error ",e);
    
}
finally{
    console.log("End of code ");
    
}